adagrams/game.py

```diff
@@ -1,11 +1,112 @@
+import random
+
 def draw_letters():
-    pass
+    """
+    build a hand of 10 letters for the user
+    """
+
+    # Letter Pool could be better in global field to easily change game
```

Comment:
Yes, this is an excellent consideration. Nice convention to declare it as a constant variable.